xwiki-platform-core/xwiki-platform-oldcore/src/test/java/com/xpn/xwiki/pdf/impl/PdfExportImplTest.java

```diff
@@ -128,6 +132,11 @@ public void applyCSSWhenExistingStyleDefinedUsingShorthandNotation() throws Exce
             + "</div>"
             + "</body></html>";
 
-        assertEquals(expected, pdfExport.applyCSS(html, css, xcontext));
+        // Depending on the configuration of the parser that is used in the test, the element content
```

Comment:
Is that not fixed (the parser used in the test)? AFAIK we control that. I'd feel better to me to have a single expectations and to have a test failure if one day the parser changes the way it handles whitespaces, so that we notice the change. 

BTW I still wonder if this parser behavior changes will lead to other problems down the road. Arguably if it does, it probably means we have a bug in the code that parses.

Reply:
> wonder if this parser behavior changes will lead to other problems

It respects the original whitespace. Less problems, probably.

Reply:
ok, let's hope it's the case ;)